src/smt/smt_engine.h

```diff
@@ -1237,6 +1237,18 @@ class CVC4_PUBLIC SmtEngine
 
   /** Container for the lemma input and output channels for this SmtEngine.*/
   LemmaChannels* d_channels;
+
+  /*---------------------------- sygus commands  ---------------------------*/
+
+  /**
+   * Set sygus conjecture is stale. This is called when new sygus constraints
```

Comment:
Please detail the whole "stale" business. It's not clear what it means neither why it is set in these cases.